src/cpu/pred/BranchPredictor.py

```diff
@@ -83,6 +85,38 @@ class BranchTargetBuffer(ClockedObject):
     numThreads = Param.Unsigned(Parent.numThreads, "Number of threads")
 
 
+class BTBIndexingPolicy(SimObject):
+    type = "BTBIndexingPolicy"
+    abstract = True
+    cxx_class = "gem5::IndexingPolicyTemplate<gem5::BTBTagType>"
+    cxx_header = "cpu/pred/btb_entry.hh"
+    cxx_template_params = ["class Types"]
```

Comment:
I copy+pasted this from the 'BaseIndexingPolicy' from 'mem/cache/tags/indexing_policies/IndexingPolicies.py' since I am trying to reuse the IndexingPolicyTemplate just like it does. Please let me know if this needs to be updated.